src/main/java/it/gov/pagopa/payhub/auth/service/exchange/ValidateExternalTokenService.java

```diff
@@ -50,6 +51,9 @@ public void validateClient(String clientId) {
     }
 
     private void validateProtocolConfiguration(String grantType, String subjectTokenType, String scope) {
+        if (subjectTokenType == null) {
```

Comment:
not just null, but also empty (eg you could use !org.springframework.util.StringUtils#hasText(java.lang.String))